TFT/src/User/Menu/PrintingMenu.c

```diff
@@ -212,15 +211,12 @@ static inline void reDrawProgress(int icon_pos)
   GUI_DispString(printinfo_points[icon_pos].x + PICON_TITLE_X, printinfo_points[icon_pos].y + PICON_TITLE_Y, (uint8_t *)buf);
   GUI_SetTextMode(GUI_TEXTMODE_NORMAL);
 
-  if (oldProgress != newProgress)  // Update the progress bar only when the progress changes
+  if (prevProgress != newProgress)  // Update the progress bar only when the progress changes
```

Comment:
No need for this conditional because `reDrawProgress()` is not called until `updatePrintProgress()` returns `true`. It returns `true` only when the progress advances.

Reply:
Got it